lib/types/poetry-manifest-type.ts

```diff
@@ -37,8 +37,7 @@ type PythonRangeVersion = string; // i.e. "~2.7"
 type Dependency = RangeVersion | DependencyObject | DependencyObject[];
 
 interface DependencyObject {
-  version: RangeVersion,
-  python?: PythonRangeVersion,
-  extras?: any
+  version: RangeVersion;
+  python?: PythonRangeVersion;
+  extras?: any;
```

Comment:
can we be more specific than `any`?

Reply:
I've taken the opportunity to remove a lot of the declarations in the interface for YAGNI reasons (and move the type definitions to sit with the code that uses them). 
If we subsequently extend functionality that requires removed attributes, they can be introduced then.